rackcity/views/it_model_views.py

```diff
@@ -390,11 +392,15 @@ def model_bulk_upload(request):
     data = JSONParser().parse(request)
     if 'import_csv' not in data:
         return JsonResponse(
-            {"failure_message": "Bulk upload request should have a parameter 'import_csv'"},
+            {"failure_message": "Bulk upload request should have a parameter 'file'"},
             status=HTTPStatus.BAD_REQUEST
         )
```

Comment:
note to self @juliasaveliff you will need to update this error message (might cause merge conflicts in my PR)